src/components/alphabet/letters-bar/letters-bar.css

```diff
@@ -0,0 +1,77 @@
+.letters-bar {
+    width: 100%;
+    height: var(--bar-size);
+    background-color: var(--color-primary);
+    box-shadow: 0 0 16px 0 black;
+    display: block;
+    font-size: calc(var(--bar-size) * 0.6);
+}
+
+.letters-list {
+    width: var(--bar-size);
+    height: 100vw;
+    margin: calc(-50vw + var(--bar-size) / 2) 0 0 calc(50vw - var(--bar-size) / 2);
+    padding: 0;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    overflow-x: hidden;
+    overflow-y: overlay;
+    transform: rotate(-90deg);
+    transition: width var(--transition-duration);
+}
+
+.letter-item {
+    width: var(--bar-size);
+    height: var(--bar-size);
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
+    align-items: center;
+    font-weight: bolder;
+    transform: rotate(90deg);
+    color: white;
+    cursor: pointer;
+}
+
+.letter-item:hover{
+    background-color: #0002;
+}
+
+.letters-list:hover::-webkit-scrollbar {
+    display: initial;
+}
+
+.letters-list::-webkit-scrollbar {
+    width: calc(var(--bar-size) * 0.15);
+    display: none;
+}
+
+.letters-list::-webkit-scrollbar-track {
+    background: #0000;
```

Comment:
its good to keep all colors stored in variables somewhere else. Then import file with all variables in specified component.